calico-enterprise/operations/cnx/access-the-manager.mdx

```diff
@@ -84,7 +84,7 @@ spec:
 
 ### Log in to the $[prodname] web console
 
-Access the $[prodname] web console in your browser at: `https://localhost:9443`
+Access the $[prodname] web console in your browser using the URL for your ingress controller. For example: `https://<ingress-url>`.
```

Comment:
@Josh-L For each of these, it would be helpful to provide guidance on how to obtain these URLs. Can you help with this?

Reply:
@ctauchen yes definitely. I'm working through these scenarios for the first time so it's a bit of a learning curve for me as well. Once I'm a bit more confident in the details and have things working I can fill you in.

Reply:
I did yes, but unfortunately I don't think the instructions are generic enough that we should be putting them in our docs. Ingress resources and loadbalancers are going to be specific to how each customer configures them so I think leaving it generic to "<ingress-url>" might be our best bet, with similar text for the loadbalancers.

I think the most important part though is that the text below the tabbed access methods needs to either be generic so that it can apply to any of the tabs above it, or it needs to also be tabbed so that it will appear correct for each tabbed method.